.travis.yml

```diff
@@ -1,10 +1,7 @@
+dist: trusty
 sudo: false
 language: python
 python:
   - "2.7"
-
 before_install: ./scripts/ci/provision.sh
-
-install: pip install -r requirements.txt --use-mirrors
```

Comment:
Will `scripts/vagrant/provision.sh` automatically be run in Travis?  If not, it seems like you would need to keep this, or change it to call that script.

Reply:
No, as I mentioned in the description, Travis runs `pip install -r requirements.txt` by default for Python projects.